src/Repository/AdherentRepository.php

```diff
@@ -64,6 +64,11 @@ public function findOneByEmail(string $email): ?Adherent
         return $this->findOneBy(['emailAddress' => $email]);
     }
 
+    public function isAdherent(string $email): bool
+    {
+        return (bool) $this->findOneBy(['emailAddress' => $email]);
```

Comment:
1) what about use the COUNT with QueryBuilder to avoid hydrate the Adherent object just for cast in boolean value?

or maybe do `return 0 < count($this->findAdherentsUuidByEmailAddress($email);` ?

2) do you need to check `adherents.adherent` value too?

Reply:
Nice catch, thx ! Alright with the COUNT :+1: